mockdata/src/main/java/org/msse/demo/mockdata/music/venue/VenueFaker.java

```diff
@@ -8,22 +8,9 @@
 
 @Service
 public class VenueFaker extends BaseFaker {
-  private static final List<String> VENUE_SUFFIX = List.of(" Arena", " Stadium", " Bandshell", " Hall", " Pavilion", " Field");
 
   public VenueFaker(Faker faker) {
     super(faker);
   }
 
-  public Venue generate(String addressId) {
-    return generate(randomId(), addressId);
-  }
-
-  public Venue generate(String venueId, String addressId) {
-    return new Venue(
-            venueId,
-            addressId,
-            faker.company().name() + faker.options().nextElement(VENUE_SUFFIX),
-            faker.number().numberBetween(5, 5000)
-    );
-  }
```

Comment:
probably don't need the venue faker anymore unless we want to keep a completely "fake" branch in the code